src/sentry/api/endpoints/project_processingissues.py

```diff
@@ -48,7 +48,8 @@ def delete(self, request: Request, project) -> Response:
         This deletes all open processing issues and triggers reprocessing if
         the user disabled the checkbox
         """
-        reprocessing_active = bool(project.get_option("sentry:reprocessing_active", True))
+        # XXX: Why does this default to `True` here?
+        reprocessing_active = bool(project.get_option(REPROCESSING_OPTION, True))
```

Comment:
isn't it defaulting to `True` because you're passing in a second parameter to `get_option`?

Reply:
Well yes, obviously :-) I was rather asking why that is? In all other parts of the codebase, it defaults to `False` instead…